src/numberApp.js

```diff
@@ -0,0 +1,55 @@
+import numberGame from './numberGame.js';
+
+const button = document.getElementById('guessButton');
+const numberInput = document.getElementById('input');
+const trysLeft = document.getElementById('trysLeft');
+const feedback = document.getElementById('feedback');
+const winLose = document.getElementById('winLose');
+let trys = 4;
+
+
+//random integer syntax pulled from MDN https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
```

Comment:
don't worry about citing MDN for referenced code, it's sort of like citing a dictionary for using a word. I like that you are thinking of that however!